addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -579,10 +533,6 @@ function factory(dependencies) {
             this.env.models['mail.message'].insert(
                 this.env.models['mail.message'].convertData(data)
             );
-            const moderationMailbox = this.env.messaging.moderation;
-            if (moderationMailbox) {
-                moderationMailbox.update({ counter: increment() });
-            }
```

Comment:
This whole method `_handleNotificationPartnerModerator` can be removed as it's not used.